CONTRIBUTING.md

```diff
@@ -96,11 +95,9 @@ The subject contains succinct description of the change:
 * no dot (.) at the end
 
 ### Body
-Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
+Wrap the body at 72 characters whenever possible (for example, don't modify backtraces to follow this rule). Just as in the **subject**, use the imperative, present tense: "change" not "changed" nor "changes".
 The body should include the motivation for the change and contrast this with previous behavior.
 
-### Footer
-The footer should contain any information about **Breaking Changes** and is also the place to
-reference GitHub issues that this commit **Closes**.
+The body contains (in order of appearance) a detailed **description** of the committed changes, references GitHub issues that the commit **closes** and any **breaking changes**. The **description** is mandatory, **closes** and **breaking changes** are optional.
 
-**Breaking Changes** should start with the word `BREAKING CHANGE:` with a space or two newlines. The rest of the commit message is then used for this.
+**Breaking changes** should start with the phrase `BREAKING CHANGE:` followed by a line break. The rest of the commit message is then used for this.
```

Comment:
Related: https://github.com/clog-tool/clog-cli/issues/81#issuecomment-175017561

As a matter of personal opinion, I think that just a single line break is worse than commit with either 2 line breaks or just a space.

e.g.

**`1`** – proposed
```
    feat: test commit
    
    BREAKING CHANGE:
    Commit with a breaking change for testing purposes.
    Rawr rawr.
```

**`2`** – currently allowed
```
    feat: test commit
    
    BREAKING CHANGE: Commit with a breaking change for testing purposes.
    Rawr rawr.
```

**`3`** – currently allowed
```
    feat: test commit
    
    BREAKING CHANGE:
    
    Commit with a breaking change for testing purposes.
    Rawr rawr.
```

`3` looks the cleanest IMO, so if there is a need to have only one "required" format, I'd prefer it to others.